tensorflow_datasets/image/__init__.py

```diff
@@ -39,3 +39,4 @@
 from tensorflow_datasets.image.quickdraw import QuickdrawBitmap
 from tensorflow_datasets.image.rock_paper_scissors import RockPaperScissors
 from tensorflow_datasets.image.svhn import SvhnCropped
+from tensorflow_datasets.image.iris import Iris
```

Comment:
Not an image dataset, move to `structured`?

Reply:
yes, I never considered that we are just looking at measurements